scripts/validate-changelog-fragments.py

```
@@ -0,0 +1,60 @@
+
+
+def validate(root: Path) -> list[str]:
+    directory = root / "changes" / "unreleased"
+    if not directory.is_dir():
+        return []
+    errors: list[str] = []
+    seen: set[int] = set()
+    for path in sorted(directory.glob("F*.yaml")):
+        text = path.read_text(encoding="utf-8", errors="ignore")
+        missing = [key for key in REQUIRED if _field(text, key) is None]
+        if missing: errors.append(f"{path}: missing {', '.join(missing)}")
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)